app/scripts/controllers/bridge/bridge-controller.ts

```diff
@@ -0,0 +1,50 @@
+import { BaseController, StateMetadata } from '@metamask/base-controller';
+import { fetchBridgeFeatureFlags } from '../../../../ui/pages/bridge/bridge.util';
+import {
+  BRIDGE_CONTROLLER_NAME,
+  DEFAULT_BRIDGE_CONTROLLER_STATE,
+} from './constants';
+import { BridgeControllerState, BridgeControllerMessenger } from './types';
+
+const metadata: StateMetadata<{ bridgeState: BridgeControllerState }> = {
+  bridgeState: {
+    persist: false,
+    anonymous: false,
+  },
+};
+
+export default class BridgeController extends BaseController<
```

Comment:
Do we want to keep this controller on the extension or should we move it to the [`core` monorepo](https://github.com/MetaMask/core/tree/main/packages)?